eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/phase0/operations/DeprecatedOperationProcessor.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.reference.phase0.operations;
+
+import tech.pegasys.teku.core.BlockProcessorUtil;
```

Comment:
do we want this processor to still be using the old `BlockProcessorUtil`?

Reply:
Yeah - this is so we can run reference tests across both the old and new logic